src/main/java/io/cdap/delta/datastream/DatastreamEventConsumer.java

```diff
@@ -252,6 +258,26 @@ private void fetchNext() throws Exception {
     }
   }
 
+  private List<SortKey> getSortKeys(GenericRecord record) {
+    GenericArray<?> keys = (GenericArray<?>) record.get(SORT_KEYS_FIELD_NAME);
+    List<SortKey> sortKeys = null;
+    if (keys != null) {
+      sortKeys = new ArrayList<>(keys.size());
+      for (Object key : keys) {
+        if (Long.class.equals(key.getClass())) {
```

Comment:
use `instanceOf` to make it consistent although `Long` is a final class

Reply:
Done